config/index.ts

```diff
@@ -47,9 +47,9 @@ export function loadConfigFromEnvironment(): boolean {
 
 export function createEmptyConfigFile(
```

Comment:
Should this util respect the `path` option? If the user specifies a custom path, they're attempting to tell us that they want to create/read a config that lives somewhere different from the default location. I think I had unintentionally broken this in my initial pass at implementing this new config behavior.

Reply:
I've fixed this behavior for the hidden config. 

We only call this function in the `init` command, and if the `useHiddenConfig` flag is used, we run `CLIConfiguration.write({ accounts: [] })`, which doesn't use the `path` option at all. Only the deprecated config respects it, which we want.